fluent/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
delete? we'll need to relicense Apache 2.0 I think unless there are objections.

Reply:
Nope, sounds good.